src/Session.ts

```diff
@@ -204,7 +209,11 @@ class Session extends EventEmitter {
       this.#transactions[transaction] = {
         resolve, reject, timeout, payload,
       };
-    }).catch(e => console.error(e));
+    }).catch(err => {
```

Comment:
Same as above regarding try/catch

Reply:
Here we have pure promise without await, so try catch will not work. Error will not be caught as we don't wait promise to resolve.